### ISSUE TITLE
cannot quiesce centos 7.3  with docker cs installed

### ISSUE THREAD

**Issue by cjpit** [vmware/open-vm-tools#127]
Hi, I'm having trouble with quisceing  using the docker commercially supported engine (1.12) latest version of vmware tools installed.  I've enabled debugging and have the vmware.log available if needed.

running open vmware tools 10.0.5-2.el7 from base repository for centos.

Thanks

ps here is what seems to be the interesting stuff

```2016-12-29T05:54:32.074Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] *** VmBackupSyncDriverStart
2016-12-29T05:54:32.089Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: Skipping remote filesystem, name=systemd-1, mntpt=/proc/sys/fs/binfmt_misc.
2016-12-29T05:54:32.090Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: Calling backend 0.
2016-12-29T05:54:32.090Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: Freezing /:/:/dev:/dev/shm:/dev/pts:/dev/hugepages:/dev/mqueue:/proc:/proc/sys/fs/binfmt_misc:/proc/fs/nfsd:/sys:/sys/kernel/security:/sys/fs/cgroup:/sys/fs/cgroup/systemd:/sys/fs/cgroup/freezer:/sys/fs/cgroup/cpuset:/sys/fs/cgroup/net_cls:/sys/fs/cgroup/cpu,cpuacct:/sys/fs/cgroup/hugetlb:/sys/fs/cgroup/blkio:/sys/fs/cgroup/memory:/sys/fs/cgroup/devices:/sys/fs/cgroup/perf_event:/sys/fs/pstore:/sys/kernel/config:/sys/fs/selinux:/sys/kernel/debug:/run:/run/user/0:/var/lib/nfs/rpc_pipefs:/backup:/data:/boot:/tmp:/var/tmp:/var/lib/docker/devicemapper:/var/lib/docker/containers/9cce731f6fd7b603742c5533f632b35566cb0c156729b9245455b73d2f25af43/shm:/var/lib/docker/containers/53a63552f23fe7657ef43c0ce908354d3856595f1f1605088436850e9dfb8a79/shm:net:[4026532656]:/var/lib/docker/containers/0ae85336050d5f2fe0c16098b3251e3ee75566f10ea6e87ca7236ac73e203805/shm:net:[4026532842]:/var/lib/docker/containers/afd75cfdea91495cdb1a0d1e026aea11bb373930bed1135cd318fc81bf6da37d/shm:net:[4026533036]:/var/lib/docker/containers/f0f1afe75fc55d6fc24706d6233cf64d06c9d9c779cdc931a2f27e0ed30a5139/shm:net:[4026533134]:/var/lib/docker/containers/732d88d980362438af9537207cea374852882326dd18b574d3daab225833e041/shm:net:[4026533322]:/var/lib/docker/containers/0313338cf33c4276f79ae577246ec857100e42098d3a9c7005e162f1a623b04b/shm:net:[4026533606]:/var/lib/docker/containers/8ba592185807134e53e0b466385c5a36d4de54fcb4bcfd04da6a57b26f938248/shm:net:[4026533700]:/var/lib/docker/containers/3cb24bfcd5ef891af1e565b484e73b203273d23e2b44ad1601d13f0232d91115/shm:net:[4026533518]:/var/lib/docker/containers/fc4aaa6342471929abd14ccdec0efa61417a023b0e159b45fcdda90f6cd1b04e/shm:net:[4026533898]:/var/lib/docker/containers/1d5f2dd98855631bf02135b24a40fc75e79810ec9cc383348390d6c4f9244432/shm:net:[4026533996]:/var/lib/docker/containers/b11c05fd21822867cc5439fce0cea2496c1655ef006f3c3f55d1b49733290a7d/shm:net:[4026534186]:/var/lib/docker/containers/154a8dfb0dfd272b3ee1670e2dd9307a0916c751bdb80c66b48f5158c1c0c22b/shm:/var/lib/docker/containers/85ad7e4597b8013c7471c9271233c268bb72e88fb48e047daa7227941bb21b30/shm:net:[4026534848]:/var/lib/docker/containers/478397a838cba04ad42fd7eff115a639ca359046249b6b1cc5e8a5902520d363/shm:net:[4026535038]:net:[4026534942]:/var/lib/docker/containers/231675caacbd8879a5190e38d9602d36bdba7f876215ae39374e2c6a5d380007/shm:/var/lib/docker/containers/ed247008de3b152c8fe1ec72d867a67b16c1e7fdc62df91898c213bf0e424f4c/shm:/var/lib/docker/containers/6374bd6c0ad7903530118a8eda0a862d39d68e112616ccd8067ee4d5bada68bb/shm:/var/lib/docker/containers/9e4999fe22aab4127a70f9fed44686423dfc296ebbea401356f7196c12313310/shm:net:[4026535320]:net:[4026535416]:net:[4026535512]:net:[4026535702]:/var/lib/docker/containers/ff694e8e87a6df3662a3b5b2ea44c6762efa0cd90955d16219bdbbaaefba2b5c/shm:/var/lib/docker/containers/c7355c89e045d80ceffa778e2747eb0e87b1f3c8004a6042220b9ad17af9052b/shm:net:[4026535898]:/var/lib/docker/containers/cdfb79f4cb74fc5e0045693bc39572164225bb673935ab1ca0d9201eab3cea4e/shm:net:[4026535996]:/var/lib/docker/containers/60138e1a8d545b25a6d5af029cf5bf4d034198d0a27a0a2fa7aa21e43e20636e/shm:net:[4026536090]:/var/lib/docker/containers/44cda39294a45a75569dfeb200f733630928775cc4c860674335702e699c6b61/shm:net:[4026536284]:net:[4026536376]:net:[4026536188]:net:[4026535544]:net:[4026536476]:/var/lib/docker/containers/71bbe7490619a237a07123d2a5fd5015ce65e84c34e1b53542a8f5a1060c4057/shm:net:[4026536574]:/var/lib/docker/containers/0f1c45b4505d660fd8cb832c1f86043b6a2837658cbce45dc7452bab325a6c3a/shm:net:[4026533430]:/var/lib/docker/containers/14333a89e53f3a667d221959808beb0d3241c67d639647737d8221f248b7ec7e/shm:net:[4026534590]:/var/lib/docker/containers/f0cd23172b061022800c559e910c5079a2ddc64c212b683bdb9a181f2803ee3e/shm:net:[4026534696] using Linux ioctls...
2016-12-29T05:54:32.090Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/'.
2016-12-29T05:54:32.091Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/'.
2016-12-29T05:54:32.111Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: successfully froze '/'.
2016-12-29T05:54:32.111Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/'.
2016-12-29T05:54:32.111Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/'.
2016-12-29T05:54:32.111Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/' returned: 16 (Device or resource busy)
2016-12-29T05:54:32.111Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/dev'.
2016-12-29T05:54:32.111Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/dev'.
2016-12-29T05:54:32.112Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/dev' returned: 95 (Operation not supported)
2016-12-29T05:54:32.112Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/dev/shm'.
2016-12-29T05:54:32.112Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/dev/shm'.
2016-12-29T05:54:32.112Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/dev/shm' returned: 95 (Operation not supported)
2016-12-29T05:54:32.112Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/dev/pts'.
2016-12-29T05:54:32.113Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/dev/pts'.
2016-12-29T05:54:32.113Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/dev/pts' returned: 95 (Operation not supported)
2016-12-29T05:54:32.113Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/dev/hugepages'.
2016-12-29T05:54:32.113Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/dev/hugepages'.
2016-12-29T05:54:32.113Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/dev/hugepages' returned: 95 (Operation not supported)
2016-12-29T05:54:32.113Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/dev/mqueue'.
2016-12-29T05:54:32.114Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/dev/mqueue'.
2016-12-29T05:54:32.114Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/dev/mqueue' returned: 95 (Operation not supported)
2016-12-29T05:54:32.114Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/proc'.
2016-12-29T05:54:32.114Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/proc'.
2016-12-29T05:54:32.114Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/proc' returned: 95 (Operation not supported)
2016-12-29T05:54:32.114Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/proc/sys/fs/binfmt_misc'.
2016-12-29T05:54:32.115Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/proc/sys/fs/binfmt_misc'.
2016-12-29T05:54:32.115Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/proc/sys/fs/binfmt_misc' returned: 95 (Operation not supported)
2016-12-29T05:54:32.115Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/proc/fs/nfsd'.
2016-12-29T05:54:32.115Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/proc/fs/nfsd'.
2016-12-29T05:54:32.116Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/proc/fs/nfsd' returned: 95 (Operation not supported)
2016-12-29T05:54:32.116Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys'.
2016-12-29T05:54:32.116Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys'.
2016-12-29T05:54:32.116Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys' returned: 95 (Operation not supported)
2016-12-29T05:54:32.116Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/kernel/security'.
2016-12-29T05:54:32.117Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/kernel/security'.
2016-12-29T05:54:32.117Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/kernel/security' returned: 95 (Operation not supported)
2016-12-29T05:54:32.117Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup'.
2016-12-29T05:54:32.117Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup'.
2016-12-29T05:54:32.117Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup' returned: 95 (Operation not supported)
2016-12-29T05:54:32.117Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/systemd'.
2016-12-29T05:54:32.118Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/systemd'.
2016-12-29T05:54:32.118Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/systemd' returned: 95 (Operation not supported)
2016-12-29T05:54:32.118Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/freezer'.
2016-12-29T05:54:32.118Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/freezer'.
2016-12-29T05:54:32.118Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/freezer' returned: 95 (Operation not supported)
2016-12-29T05:54:32.118Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/cpuset'.
2016-12-29T05:54:32.118Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/cpuset'.
2016-12-29T05:54:32.119Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/cpuset' returned: 95 (Operation not supported)
2016-12-29T05:54:32.119Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/net_cls'.
2016-12-29T05:54:32.119Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/net_cls'.
2016-12-29T05:54:32.119Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/net_cls' returned: 95 (Operation not supported)
2016-12-29T05:54:32.119Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/cpu,cpuacct'.
2016-12-29T05:54:32.119Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/cpu,cpuacct'.
2016-12-29T05:54:32.119Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/cpu,cpuacct' returned: 95 (Operation not supported)
2016-12-29T05:54:32.120Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/hugetlb'.
2016-12-29T05:54:32.120Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/hugetlb'.
2016-12-29T05:54:32.120Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/hugetlb' returned: 95 (Operation not supported)
2016-12-29T05:54:32.120Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/blkio'.
2016-12-29T05:54:32.120Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/blkio'.
2016-12-29T05:54:32.120Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/blkio' returned: 95 (Operation not supported)
2016-12-29T05:54:32.120Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/memory'.
2016-12-29T05:54:32.121Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/memory'.
2016-12-29T05:54:32.121Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/memory' returned: 95 (Operation not supported)
2016-12-29T05:54:32.121Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/devices'.
2016-12-29T05:54:32.121Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/devices'.
2016-12-29T05:54:32.121Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/devices' returned: 95 (Operation not supported)
2016-12-29T05:54:32.121Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/cgroup/perf_event'.
2016-12-29T05:54:32.121Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/cgroup/perf_event'.
2016-12-29T05:54:32.122Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/cgroup/perf_event' returned: 95 (Operation not supported)
2016-12-29T05:54:32.122Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/pstore'.
2016-12-29T05:54:32.122Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/pstore'.
2016-12-29T05:54:32.122Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/pstore' returned: 95 (Operation not supported)
2016-12-29T05:54:32.122Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/kernel/config'.
2016-12-29T05:54:32.123Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/kernel/config'.
2016-12-29T05:54:32.123Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/kernel/config' returned: 95 (Operation not supported)
2016-12-29T05:54:32.123Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/fs/selinux'.
2016-12-29T05:54:32.123Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/fs/selinux'.
2016-12-29T05:54:32.123Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/fs/selinux' returned: 95 (Operation not supported)
2016-12-29T05:54:32.123Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/sys/kernel/debug'.
2016-12-29T05:54:32.124Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/sys/kernel/debug'.
2016-12-29T05:54:32.124Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/sys/kernel/debug' returned: 95 (Operation not supported)
2016-12-29T05:54:32.124Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/run'.
2016-12-29T05:54:32.124Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/run'.
2016-12-29T05:54:32.124Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/run' returned: 95 (Operation not supported)
2016-12-29T05:54:32.124Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/run/user/0'.
2016-12-29T05:54:32.125Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/run/user/0'.
2016-12-29T05:54:32.125Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/run/user/0' returned: 95 (Operation not supported)
2016-12-29T05:54:32.125Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/var/lib/nfs/rpc_pipefs'.
2016-12-29T05:54:32.126Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/var/lib/nfs/rpc_pipefs'.
2016-12-29T05:54:32.126Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/var/lib/nfs/rpc_pipefs' returned: 95 (Operation not supported)
2016-12-29T05:54:32.126Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/backup'.
2016-12-29T05:54:32.126Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/backup'.
2016-12-29T05:54:32.130Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: successfully froze '/backup'.
2016-12-29T05:54:32.130Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/data'.
2016-12-29T05:54:32.130Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/data'.
2016-12-29T05:54:32.178Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: successfully froze '/data'.
2016-12-29T05:54:32.179Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/boot'.
2016-12-29T05:54:32.179Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/boot'.
2016-12-29T05:54:32.182Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: successfully froze '/boot'.
2016-12-29T05:54:32.183Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/tmp'.
2016-12-29T05:54:32.183Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/tmp'.
2016-12-29T05:54:32.184Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/tmp' returned: 16 (Device or resource busy)
2016-12-29T05:54:32.184Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/var/tmp'.
2016-12-29T05:54:32.184Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/var/tmp'.
2016-12-29T05:54:32.185Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/var/tmp' returned: 16 (Device or resource busy)
2016-12-29T05:54:32.185Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/var/lib/docker/devicemapper'.
2016-12-29T05:54:32.185Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/var/lib/docker/devicemapper'.
2016-12-29T05:54:32.186Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/var/lib/docker/devicemapper' returned: 16 (Device or resource busy)
2016-12-29T05:54:32.186Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/var/lib/docker/containers/9cce731f6fd7b603742c5533f632b35566cb0c156729b9245455b73d2f25af43/shm'.
2016-12-29T05:54:32.186Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/var/lib/docker/containers/9cce731f6fd7b603742c5533f632b35566cb0c156729b9245455b73d2f25af43/shm'.
2016-12-29T05:54:32.187Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/var/lib/docker/containers/9cce731f6fd7b603742c5533f632b35566cb0c156729b9245455b73d2f25af43/shm' returned: 95 (Operation not supported)
2016-12-29T05:54:32.187Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path '/var/lib/docker/containers/53a63552f23fe7657ef43c0ce908354d3856595f1f1605088436850e9dfb8a79/shm'.
2016-12-29T05:54:32.187Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freezing path '/var/lib/docker/containers/53a63552f23fe7657ef43c0ce908354d3856595f1f1605088436850e9dfb8a79/shm'.
2016-12-29T05:54:32.188Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: freeze on '/var/lib/docker/containers/53a63552f23fe7657ef43c0ce908354d3856595f1f1605088436850e9dfb8a79/shm' returned: 95 (Operation not supported)
2016-12-29T05:54:32.188Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: opening path 'net'.
2016-12-29T05:54:32.188Z| vcpu-7| I125: Guest: [   debug] [vmsvc:vmsvc] SyncDriver: failed to open 'net': 2 (No such file or directory)
2016-12-29T05:54:32.189Z| vmx| I125: Guest: [ warning] [vmsvc:vmbackup] Error freezing filesystems.
6-12-29T05:54:32.675Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] RpcIn: received 24 bytes, content:"Set_Option broadcastIP 1"
2016-12-29T05:54:32.676Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmtoolsd] Setting option 'broadcastIP' to '1'.
2016-12-29T05:54:32.678Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] RpcChannel: Sending: 33 bytes
2016-12-29T05:54:32.678Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] VSockChan: Sending request for conn 8,  reqLen=33
2016-12-29T05:54:32.679Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] SimpleSock: Sent 65 bytes from socket 8
2016-12-29T05:54:32.679Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] SimpleSock: Recved 4 bytes from socket 8
2016-12-29T05:54:32.679Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] SimpleSock: Recved 14 bytes from socket 8
2016-12-29T05:54:32.680Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] VSockOut: recved 2 bytes for conn 8
2016-12-29T05:54:32.680Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] RpcChannel: Recved 0 bytes
2016-12-29T05:54:32.681Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] RpcIn: sending 3 bytes
2016-12-29T05:54:33.075Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] *** VmBackupAsyncCallback
2016-12-29T05:54:33.086Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] Async request 'VmBackupSyncDriverStart' completed
2016-12-29T05:54:33.098Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] *** VmBackupSyncDriverReadyForSnapshot
2016-12-29T05:54:33.458Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] *** VmBackupKeepAliveCallback
2016-12-29T05:54:33.473Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] *** VmBackup_SendEvent
2016-12-29T05:54:33.485Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] Sending vmbackup event: vmbackup.eventSet req.keepAlive 0 
2016-12-29T05:54:33.500Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] RpcChannel: Sending: 35 bytes
2016-12-29T05:54:33.500Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] VSockChan: Sending request for conn 8,  reqLen=35
2016-12-29T05:54:33.501Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] SimpleSock: Sent 67 bytes from socket 8
2016-12-29T05:54:33.501Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] SimpleSock: Recved 4 bytes from socket 8
2016-12-29T05:54:33.502Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] SimpleSock: Recved 14 bytes from socket 8
2016-12-29T05:54:33.502Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] VSockOut: recved 2 bytes for conn 8
2016-12-29T05:54:33.502Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] RpcChannel: Recved 0 bytes
2016-12-29T05:54:34.117Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] *** VmBackupAsyncCallback
2016-12-29T05:54:34.129Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] *** VmBackupEnableSync
2016-12-29T05:54:34.141Z| vcpu-6| I125: Guest: [    info] [vmsvc:guestinfo] New value for stats-interval is 20s.
2016-12-29T05:54:34.142Z| vcpu-6| I125: Guest: [    info] [vmsvc:guestinfo] New value for poll-interval is 30s.
2016-12-29T05:54:34.142Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] Flushed 0 log messages from cache after resuming log IO.
2016-12-29T05:54:34.143Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] *** VmBackup_SendEvent
2016-12-29T05:54:34.153Z| vmx| I125: Guest: [   debug] [vmsvc:vmbackup] Sending vmbackup event: vmbackup.eventSet req.error 3 Error when enabling the sync provider.
2016-12-29T05:54:34.165Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] RpcChannel: Sending: 69 bytes
2016-12-29T05:54:34.165Z| vcpu-6| I125: Guest: [   debug] [vmsvc:vmsvc] VSockChan: Sending request for conn 8,  reqLen=69
2016-12-29T05:54:34.166Z| vmx| I125: Msg_Post: Warning
2016-12-29T05:54:34.166Z| vmx| I125: [msg.snapshot.quiesce.vmerr] The guest OS has reported an error during quiescing.
2016-12-29T05:54:34.166Z| vmx| I125+ The error code was: 3
2016-12-29T05:54:34.166Z| vmx| I125+ The error message was: Error when enabling the sync provider.

```

**Comment by cjpit** [comment]
I raised this directly with vmware support as well - Supposedly is fixed in version 10.5.1 whenever that comes out...

**Comment by ravindravmw** [comment]
Thanks for reporting this. We are aware of this issue, a change in docker behavior seems to have caused this and a fix is in progress. I can't comment on versioning because release versions are subject to change.

**Comment by ravindravmw** [comment]
Fix for this has been released in open-vm-tools 10.1.5.

**Comment by odg0318** [comment]
@ravindravmw Is this issue really resolved?
I installed `10.1.10.63510` on my vm instance.
```
$ vmtoolsd -v
VMWare Tools daemon, version 10.1.10.63510 (build-6082533)
```
It is resolved that vm becomes unavailable to access but docker container cannot be removed with `device or resource` message after backup is executed.
Docker version is `17.12.0-ce` and storage driver is `devicemapper`.
`ntpd` process holds docker's volumes.
